### ISSUE TITLE
Nightly s390 manywheel-py3_8-cpu-s390x-build is failing

### ISSUE THREAD

**Issue by atalman** [pytorch/pytorch#127568]
### 🐛 Describe the bug

Here is the failure:
https://github.com/pytorch/pytorch/actions/runs/9298772363/job/25591314688

```
-- Could NOT find Python (missing: Python_INCLUDE_DIRS Python_LIBRARIES Python_NumPy_INCLUDE_DIRS Interpreter Development NumPy Development.Module Development.Embed) 
CMake Error at cmake/Dependencies.cmake:874 (message):
  Python development libraries could not be found.
Call Stack (most recent call first):
  CMakeLists.txt:852 (include)
```

### Versions

2.4.0

cc @seemethere @malfet @osalpekar

**Comment by seemethere** [comment]
Can someone on the IBM team take a look at this? @atalman can you ping them here. 

We should probably turn this build off if it continues to fail after a week.

**Comment by Skylion007** [comment]
This likely related to the changes @cyyever has been making to the cmake build system.

**Comment by cyyever** [comment]
I am submitting a PR to fix it

**Comment by Skylion007** [comment]
Closed by #127600
